features/FEATURE_LWIP/lwip-interface/lwip-sys/arch/cc.h

```diff
@@ -96,6 +75,8 @@ typedef uintptr_t          mem_ptr_t;
        doesn't generate compiler warning */
     #define LWIP_CHKSUM_ALGORITHM   0
 
+    #include <stdint.h>
+    typedef uint16_t           u16_t;
```

Comment:
And actually, don't you still need stddef.h for size_t? Put those two back at the top where they were?